modules/standard/Map.chpl

```diff
@@ -157,6 +155,7 @@ module Map {
       :arg parSafe: If `true`, this map will use parallel safe operations.
       :type parSafe: bool
     */
+    pragma "unsafe"
```

Comment:
> Initializing a map from another map of owned would require removing them from other before adding them to this, which isn't a really well defined operation yet.

I guess it's not clear to me why that's not a well defined operation? This function would completely clear out all the elements in a map of owned. Of course we might think that's not good API design but we could implement it.

It would be OK with me personally if we didn't allow `init=` from maps with key/value types that aren't returning true for `isCopyableType(t)`.

Reply:
Maybe instead of not well defined I should say "there's no precedent for it in the language". It's not something we've ever done before, and I think it goes against the expected behavior for `init=`, assignment, etc.